vtr_flow/benchmarks/noc/Readme.txt

```diff
@@ -0,0 +1,44 @@
+###################################################
+# NoC Benchmark Design Files
+###################################################
+
+This directory contains sample benchmark design files that incorporate
+and embedded Network-on-Chip (NoC). 
+
+Each benchmark design consists of a design file (.blif circuit netlist) and a 
+noc traffic flows file (.flows file). The names of the two files for
+each benchmark design are the same. For example, below is an example of two
+files for a sample benchmark design.
+
+test_benchmark_circuit:
+	test_benchmark_circuit.blif
+	test_benchmark_circuit.flows
+	
+
+# Test Benchmark Designs
+
+The benchmark designs found under the "Test_Designs" folder are relatively
+simple designs that are mainly used to verify components NoC flow in the VPR software.
+Whenever a new or existing feature needs to be tested, the benchmark designs found here
+should be used.
+
+
+# Synthetic Benchmark Designs
+
+The benchmark designs found under the "Synthetic_Designs" folder are small to medium
+sized designs that are used verify the correctness of the VPR software. These designs
+are manually created to guarantee and optimal reference solution. By running these designs
```

Comment:
created to have understandable structures -- e.g. a mesh with traffic flows to nearest neighbours where we know the optimal solution for NoC traffic minimization.

Reply:
Done